web/src/auth/LoginPage.js

```diff
@@ -389,6 +418,12 @@ class LoginPage extends React.Component {
         this.checkCaptchaStatus(values);
         return;
       }
+      const application = this.getApplicationObj();
```

Comment:
what will happen if `this.state.enableCaptchaModal===CaptchaRule.Always` or `this.state.enableCaptchaModal===CaptchaRule.Dynamic`